articles/logs/index.md

```diff
@@ -4,39 +4,107 @@ url: /logs
+
+## Log Data Event Listing
+
+The following table lists the codes associated with the appropriate log events.
+
+<table>
+  <thead><tr><th>Event Code</th><th>Event</th></tr></thead>
+  <tbody>
+  <tr><td>admin_update_launch</td><td>Auth0 Update Launched</td></tr>
+  <tr><td>api_limit</td><td>Rate Limit On API</td></tr>
+  <tr><td>cls</td><td>Code/Link Sent</td></tr>
+  <tr><td>coff</td><td>Connector Offline</td></tr>
+  <tr><td>con</td><td>Connector Online</td></tr>
+  <tr><td>cs</td><td>Code Sent</td></tr>
+  <tr><td>du</td><td>Deleted User</td></tr>
+  <tr><td>f</td><td>Failed Login</td></tr>
+  <tr><td>fapi</td><td>Failed API Operation</td></tr>
+  <tr><td>fc</td><td>Failed by Connector</td></tr>
+  <tr><td>fce</td><td>Failed Change Email</td></tr>
+  <tr><td>fco</td><td>Failed by CORS</td></tr>
+  <tr><td>fcp</td><td>Failed Change Password</td></tr>
+  <tr><td>fcpn</td><td>Failed Change Phone Number</td></tr>
+  <tr><td>fcpr</td><td>Failed Change Password Request</td></tr>
+  <tr><td>fcpro</td><td>Failed Connector Provisioning</td></tr>
+  <tr><td>fcu</td><td>Failed Change Username</td></tr>
+  <tr><td>fdu</td><td>Failed User Deletion</td></tr>
+  <tr><td>feacft</td><td>Failed Exchange</td></tr>
+  <tr><td>fn</td><td>Failed Sending Notification</td></tr>
+  <tr><td>fp</td><td>Failed Login (Incorrect Password)</td></tr>
+  <tr><td>fs</td><td>Failed Signup</td></tr>
+  <tr><td>fu</td><td>Failed Login (Invalid Email/Username)</td></tr>
+  <tr><td>fv</td><td>Failed Verification Email</td></tr>
+  <tr><td>fvr</td><td>Failed Verification Email Request</td></tr>
+  <tr><td>limit_mu</td><td>Blocked IP Address</td></tr>
+  <tr><td>limit_ui</td><td>Too Many Calls to /userinfo Endpoint</td></tr>
+  <tr><td>limit_wc</td><td>Blocked Account</td></tr>
+  <tr><td>s</td><td>Success Login</td></tr>
+  <tr><td>sapi</td><td>Success API Operation</td></tr>
+  <tr><td>sce</td><td>Success Change Email</td></tr>
+  <tr><td>scp</td><td>Success Change Password</td></tr>
+  <tr><td>scpn</td><td>Success Change Phone Number</td></tr>
+  <tr><td>scpr</td><td>Success Change Password Request</td></tr>
+  <tr><td>scu</td><td>Success Change Username</td></tr>
+  <tr><td>sdu</td><td>Success User Deletion</td></tr>
+  <tr><td>seacft</td><td>Success Exchange</td></tr>
+  <tr><td>ss</td><td>Success Signup</td></tr>
+  <tr><td>sv</td><td>Success Verification Email</td></tr>
+  <tr><td>svr</td><td>Success Verification Email Request</td></tr>
+  <tr><td>sys_os_update_end</td><td>Auth0 OS Update Ended</td></tr>
+  <tr><td>sys_os_update_start</td><td>Auth0 OS Update Started</td></tr>
+  <tr><td>sys_update_end</td><td>Auth0 Update Ended</td></tr>
+  <tr><td>sys_update_start</td><td>Auth0 Update Started</td></tr>
+  <tr><td>w</td><td>Warnings During Login</td></tr>
+  </tbody>
+</table>
+
+## External Resources
+
+### Useful Links for Integrating Auth0 with Splunk
+
+* [Sending events to Splunk from Auth0](/scenarios/splunk)
```

Comment:
Hey @khorne3 why do we link to that doc? Isn't it outdated? I get the impression that this is how we did the splunk integration before the extension. Shouldn't we link to the [Auth0 Logs to Splunk doc](https://auth0.com/docs/extensions/splunk) instead?